libraries/AP_HAL_ChibiOS/hwdef/Aocoda-RC-H743Dual/hwdef.dat

```diff
@@ -0,0 +1,218 @@
+# hw definition file for processing by chibios_pins.py
+# for Aocoda-RC-H743Dual: MPU6000/BMI270x2
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# board ID for firmware load
+APJ_BOARD_ID AP_HW_AOCODA-RC-H743DUAL
+
+# crystal frequency, setup to use external oscillator
+OSCILLATOR_HZ 8000000
+
+# flash size
+FLASH_SIZE_KB 2048
+
+env OPTIMIZE -Os
+
+# bootloader takes first sector
+FLASH_RESERVE_START_KB 128
+
+# ChibiOS system timer
+STM32_ST_USE_TIMER 12
+define CH_CFG_ST_RESOLUTION 16
+
+# USB setup
+USB_STRING_MANUFACTURER "Aocoda-RC-H743Dual"
+
+# Now we define the pins that USB is connected on.
+PA11 OTG_FS_DM OTG1
+PA12 OTG_FS_DP OTG1
+
+# OTG USB Detect
+PE2 EXT_CS2 CS

```

Comment:
Isn't this usually VBUS?

Reply:
It's VBUS actually. I'm NOT sure if the configuration should be this way, just reference Matek H743


https://github.com/ArduPilot/ardupilot/blob/7d4d2f98bf98512e645288982961bd5aad61194a/libraries/AP_HAL_ChibiOS/hwdef/MatekH743/hwdef.dat#L48

Reply:
Or should I configure this as below?

```
PE2 VBUS INPUT
```

Reply:
right, that's what I was thinking

Reply:
Will do

Reply:
not changed yet,fyi

Reply:
I have checked again (it should be update 9 hours ago, but don't know why). Can you help to check it again.